br.app.pw3270.yml

```diff
@@ -0,0 +1,53 @@
+app-id: br.app.pw3270
```

Comment:
`app.br` doesn't exist.

The app-id doesn't meet the requirements. 
See https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
It can be br.app.pw3270.terminal to avoid redundancy?

Reply:
The doc says:

> "Don’t hesitate to repeat the application name even if it already is part of the domain name section of the identifier "

There is nothing at pw3270.app.br, even though the DNS answers.

Reply:
Yes. The domain was just registered. I thought the domain just needed to exist, it doesn't have a homepage yet.

Reply:
you could have just use github for the app id.

it you tell me it's yours it's fine. Just making sure here.

Reply:
I prefer to use br.app.pw3270.terminal because I've plans to add the ipc module in the future and this module own a d-bus name for evere active session just adding the session name, for example, the first session will be br.app.pw3270.terminal.a, the second one br.app.pw3270.terminal.b this names seens more user friendly to me.

Reply:
register the domain is the solution

Reply:
> register the domain is the solution pw3270.app.br

Already did, take a look at http://pw3270.app.br/